tests/Propulsion.Tests/AppendsEpochTests.fs

```diff
@@ -25,5 +26,43 @@ let ``residual span shouldn't be affected by earlier events in closed spans`` ()
 
     test <@ residual1 = spans3 && accepted2 = (spans3 |> Array.map (fun {p = p} -> p)) && residual2 = [||] @>
 
+[<Fact>]
+let ``Already ingested events should be removed by ingestion on closed epoch`` () =
+    let shouldClose = (<) 10
+    let spans1: Events.StreamSpan array = [| { p = IndexStreamId.ofP "Cat-Id"; i = 0L; c = [| "a"; "a" |] } |]
```

Comment:
I erroneously switched to using `array` in the codebase for a while. The F# style guide says to use `[]` and am trying to do that for all code going forward (I think the repo is compliant too)
(Appreciate the fact the formatting is otherwise perfect)
```suggestion
    let spans1: Events.StreamSpan[] = [| { p = IndexStreamId.ofP "Cat-Id"; i = 0L; c = [| "a"; "a" |] } |]
```